Assets/Car EMG/Scripts/BCIMenu.cs

```diff
@@ -101,12 +101,30 @@ void Update()
 
     public void SetThresholdBar(int slider)
     {
-        bciReader.SetThreshold(slider, zeroSlider.value);
+        if (slider >= 0 && slider <= 1)
```

Comment:
This should be checking that 0 <= zeroSlider.value <= 1, not 0 <= slider <= 1. My bad, unclear specifications.